argus/environment.py

```diff
@@ -157,3 +217,22 @@ def _wait_for_api():
                 break
             except Exception:
                 time.sleep(1)
+
+    def _get_services(self):
+        services = self._run_command("systemctl -a")

```

Comment:
I'm not keen on hardcoding systemctl here. Here's a solution: put these external commands in the environment configuration and reuse them from here. This will need some changes though.